src/app/models/weather-alert.model.ts

```diff
@@ -23,10 +23,10 @@ export class WeatherAlertResponse {
       if (this.flowStatus === 'SUCCESS') {
         const tempAllActiveWeatherAlerts: WeatherAlertInfo[] = this.parseApiResponse(apiResponse);
         this.allActiveWeatherAlerts = this.filterDuplicateAlerts(tempAllActiveWeatherAlerts);
-        this.activeAlertDescription = this.descriptionForToast(this.allActiveWeatherAlerts);
         // the below variables are set to 0 index of array because that will be up-to-date active alert
-        this.activeAlertEvent = this.allActiveWeatherAlerts && this.allActiveWeatherAlerts[0] && this.allActiveWeatherAlerts[0].event ? this.allActiveWeatherAlerts[0].event : '';
         this.activeAlertHeadline = this.allActiveWeatherAlerts && this.allActiveWeatherAlerts[0] && this.allActiveWeatherAlerts[0].headline ? this.allActiveWeatherAlerts[0].headline : '';
+        this.activeAlertDescription = this.allActiveWeatherAlerts && this.allActiveWeatherAlerts[0] && this.allActiveWeatherAlerts[0].description ? this.allActiveWeatherAlerts[0].description : '';
```

Comment:
What do you think about redoing this section here to have

`const weatherAlertValid = this.allActiveWeatherAlerts && this.allActiveWeatherAlerts[0];`

And then for each line we could do:

`this.activeAlertDescription = weatherAlertValid && weatherAlertValid.description ? weatherAlertValid.description : '';`